container.go

```diff
@@ -90,7 +91,8 @@ type ContainerRequest struct {
 	ReaperImage     string              // alternative reaper image
 	AutoRemove      bool                // if set to true, the container will be removed from the host when stopped
 	NetworkMode     container.NetworkMode
-	AlwaysPullImage bool // Always pull image
+	AlwaysPullImage bool               // Always pull image
+	BuildArgs       map[string]*string // for specifying network aliases
```

Comment:
Hey! Thank you for your work! I think this fits better as part of the `FromDockerfile` what do you think?

Reply:
For sure! I've changed to there.